src/dso_api/dynamic_api/fields.py

```diff
@@ -30,7 +30,9 @@ def get_url(self, obj, view_name, request, format=None):
             lookup_value, version = split_on_separator(obj.pk)
             kwargs = {self.lookup_field: lookup_value}
 
-            base_url = self.reverse(view_name, kwargs=kwargs, request=request, format=format)
+            base_url = self.reverse(
```

Comment:
Weer regellengte gedoe met Black?

Reply:
> Weer regellengte gedoe met Black?

Vreemd. Ik heb dus gerebased op master. Daar zit nu de commithook is en de regellengte staat ingesteld op 99. Dus de pre-commit hook zou dit afgevangen moeten hebben. Ik snap het niet helemaal.

Reply:
Ik ga kijken wat er aan de hand is. Wellicht kan ik het corrigeren en de changes squashen.

Reply:
Ik heb nu de hele repo geblackened met `pre-commit run --all-file`. Toen zag ik idd weer wat aanpassingen aan regellengtes. Die wijzigingen heb ik gesquashed in de eerste commit. Nu lijkt het beter.